mycroft/skills/main.py

```diff
@@ -290,8 +307,12 @@ def download_skills(self, speak=False):
         default_names = set(chain(default_groups['default'], platform_groups))
         default_skill_errored = False
 
+        skills_data = self.load_skills_data()
+
         def install_or_update(skill):
             """Install missing defaults and update existing skills"""
+            if skills_data.get(skill.name, {}).get('beta'):
+                skill.sha = None
```

Comment:
I assume this is what makes it fetch the head of the 18.02 branch? Maybe add a comment since it's non-obvious

Reply:
I realized I could just change it to `skill.sha = 'HEAD'`. Let me know if you think a comment would still be nice.

Reply:
Nah, that's readable enough. will retest